shortstories/views.py

```diff
@@ -9,3 +12,14 @@ def index(request):
 def detail(request, shortstory_id):
     shortstory =  get_object_or_404(ShortStory, pk=shortstory_id)
     return render(request,'shortstories/detail.html',{'shortstory':shortstory})
+
+def create(request):
+    if request.method == "POST":
+       form = ShortStoryForm(request.POST)
+       if form.is_valid():
```

Comment:
do you need an else for this `if`?

Reply:
no, as it runs statements  when form is valid and else shows errors we defined in template